pkg/authorization/authorizer/scope/converter.go

```diff
@@ -177,7 +177,6 @@ func (userEvaluator) ResolveRules(scope, namespace string, clusterPolicyGetter c
 	case UserAccessCheck:
 		return []authorizationapi.PolicyRule{
 			authorizationapi.NewRule("create").Groups(kauthorizationapi.GroupName).Resources("selfsubjectaccessreviews").RuleOrDie(),
-			{Verbs: sets.NewString("create"), APIGroups: []string{authorizationapi.GroupName, authorizationapi.LegacyGroupName}, Resources: sets.NewString("subjectaccessreviews", "localsubjectaccessreviews"), AttributeRestrictions: &authorizationapi.IsPersonalSubjectAccessReview{}},
```

Comment:
This change broke the ability of a client to request a subject access review without scopes.  Scopes aren't in upstream Kube, and as a result, you can no longer use a token that only has "user:info user:check-access" scopes to do a SAR that tells you what the user can do.  Which breaks the ability to use this as a proxy.

If I can't do SAR and take scopes into account, then we are regressing functionality and shouldn't remove this until we have an equivalent behavior in Kube.  If there's a workaround I don't see it - couldn't find anything that allowed a separate channel for passing scopes.